### PR TITLE
TAXSIM27 validation updates [Review]

### PR DESCRIPTION
**Pull request by chusloj**
This PR updates the `README.md` to provide step-by-step instructions for comparing Tax Calculator and TAXSIM27, and also updates the `taxsim_input.py` file.

In `taxsim_input.py`, the `random_integers` `numpy` function is deprecated, so it was changed to `randint`. `taxsim_input.py` was edited to avoid errors thrown when `randint` attempts to generate an integer between 0 and 0.

@Peter-Metz

### CODE REVIEW

File: taxcalc/validation/taxsim27/README.md [hunk 1]
```diff
@@ -36,6 +36,17 @@ In the following results, when we say "same results" we mean that the
 federal individual income tax liabilities and payroll tax liabilities
 being compared have differences of no larger than one cent.
 
+
+Instructions
+------------------
+
+1. Use `taxsim_input.py` to create the `.in` files necessary to input into TAXSIM.
```
Comment (Peter-Metz) [inline taxcalc/validation/taxsim27/README.md:43]: Might be helpful to specify TAXSIM-27 throughout

File: taxcalc/validation/taxsim27/README.md [hunk 2]
```diff
@@ -36,6 +36,17 @@ In the following results, when we say "same results" we mean that the
...
+2. Head to the [TAXSIM website](http://users.nber.org/~taxsim/taxsim27/) and under _Upload a file_, place each of the `.in` files in and save the outputs as `LYY.in.out-taxsim`. When you run the TAXSIM simulation, make sure to switch _Show detailed intermediate calculations_ to _on_.
+3. Open each of these `.in.out-taxsim` filed and remove the first few lines containing the variable names.
```
Comment (Peter-Metz) [inline taxcalc/validation/taxsim27/README.md:45]: filed --> files

File: taxcalc/validation/taxsim27/README.md [hunk 3]
```diff
@@ -36,6 +36,17 @@ In the following results, when we say "same results" we mean that the
...
+4. Compress these files in a `.zip` format (make sure to compress the files themselves and **do not** place the files in a folder and compress the folder).
+5. Go to the `validation` directory above the directory this `README` is housed in and run the shell script `tests.sh`.
```
Comment (Peter-Metz) [inline taxcalc/validation/taxsim27/README.md:47]: "Navigate to `taxcalc/validation` and run `tests.sh`"

**Comment by Peter-Metz** [comment]
Thanks for the PR @chusloj!

Two comments:
- Looking at the documentation of `np.random_integers` and `np.randint`, it looks like the former includes its upper/lower bounds while the latter includes just the lower bound. Is this being accounted for anywhere? I think once we account for this difference, we won't need the conditional logic with the zeros any more.
- Looks like Travis is failing because of a PEP8 issue in `taxsim_input.py`. You can see what the specific issue is locally by running `pycodestyle taxcalc` in the top-level directory

**Comment by chusloj** [comment]
Thanks for the detailed review @Peter-Metz,

Per our discussion, the upper bound of each `randint` function within `taxsim_input.py` was increased by 1 to make the function effectively generate random integers on a closed interval. Also, I modified the `taxsim_input.py` file from the `master` branch, so the file follows PEP8 conventions.

**Comment by chusloj** [comment]
> Travis is failing because of a PEP8 issue in `taxsim_input.py`

Code style issues fixed and tested.

**Comment by MattHJensen** [comment]
@chusloj, thanks a lot for this PR, and @Peter-Metz, thanks for your review. Merging.

**Pull request merged**